services/src/main/java/io/druid/cli/CliCoordinator.java

```diff
@@ -155,6 +172,25 @@ public void configure(Binder binder)
             LifecycleModule.register(binder, Server.class);
             LifecycleModule.register(binder, DatasourcesResource.class);
 
+            ConditionalMultibind.create(
+                properties,
+                binder,
+                DruidCoordinatorHelper.class,
+                CoordinatorIndexingServiceHelper.class
+            ).addConditionBinding(
+                "druid.coordinator.merge.strategy",
+                Predicates.equalTo("append"),
+                DruidCoordinatorSegmentMerger.class
+            ).addConditionBinding(
+                "druid.coordinator.conversion.on",
+                Predicates.equalTo("true"),
+                DruidCoordinatorVersionConverter.class
+            ).addConditionBinding(
+                "druid.coordinator.kill.on",
```

Comment:
by default it is "false", it was set [here](https://github.com/druid-io/druid/blob/master/server/src/main/java/io/druid/server/coordinator/DruidCoordinatorConfig.java#L57)